exec/java-exec/src/test/java/org/apache/drill/test/rowSet/test/TestSchemaBuilder.java

```diff
@@ -81,7 +79,7 @@ public void testRowPreBuilt() {
 
     MaterializedField aField = MaterializedField.create("a",
         Types.optional(MinorType.VARCHAR));
-    AbstractColumnMetadata bCol = MetadataUtils.newScalar("b",
+    ColumnMetadata bCol = MetadataUtils.newScalar("b",
```

Comment:
Consider adding test functions for the new stand-alone versions of the builders.